test/e2e/common/node/sysctl.go

```diff
@@ -148,7 +148,7 @@ var _ = SIGDescribe("Sysctls [LinuxOnly] [NodeConformance]", func() {
 		client := f.ClientSet.CoreV1().Pods(f.Namespace.Name)
 		_, err := client.Create(ctx, pod, metav1.CreateOptions{})
 
-		gomega.Expect(err).NotTo(gomega.BeNil())
+		framework.ExpectNoError(err)
```

Comment:
```suggestion
		framework.ExpectError(err)
```

Reply:
Right, will fix. However, my plan is to get rid of all useless framework gomega wrappers including `ExpectError`, so I'll use `gomega.Expect(err).To(gomega.HaveOccurred())`.

Reply:
Even then it's not a good assertion: instead of passing when *some* error occurs, it should be more specific and check that exactly *the* expected error occurs.

Reply:
It's okay here, such stricter checking follows.

Reply:
> I'll use gomega.Expect(err).To(gomega.HaveOccurred()).

`hack/verify-test-code.sh` objected against that. I'll switch to it later.

Reply:
I just suggested the contrary of the library, but feel free to choose the one you think is best

Reply:
I switched again, now to what was suggested in https://github.com/onsi/gomega/issues/642#issuecomment-1440344286... :sweat_smile: